bin/utils/node-utils.sh

```diff
@@ -149,12 +151,9 @@ check_node_version() {
   
   too_low=""
   too_low_support=""
-  if [[ ${node_major_version} -lt 8 ]]
+  if [[ ${node_major_version} -lt ${minVersion} ]]
   then
     too_low="true"
-  elif [[ ${node_major_version} -eq 8 ]] && [[ ${current_year} -gt 2021 ]]
-  then
-    too_low_support="true"
   fi
 
   if [[ ${too_low} == "true" ]]
```

Comment:
this section below has wrong text. there's 3 cases below of errors.
instead, just do:

if [[ ${too_low} == "true" ]]
then
  log_message "Warning: Zowe no longer supports using NodeJS versions lower than ${minVersion}, please upgrade NodeJS"
fi